### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

**Comment by notriddle** [comment]
@Eijebong

You want to review this one?

File: src/lib.rs [hunk 1]
```diff
@@ -90,14 +92,42 @@ pub struct Ammonia<'a> {
...
     fn default() -> Self {
         let tags = hashset![
-            "a", "b", "blockquote", "br", "code", "dd", "del", "dl", "dt",
-            "em", "i", "h1", "h2", "h3", "hr", "img", "kbd", "li", "ol", "p",
-            "pre", "s", "strike", "strong", "sub", "sup", "table", "tbody",
-            "td", "th", "thead", "tr", "ul", "hr"
-        ];
-        let generic_attributes = hashset![
-            "title"
+            "a",
```
Comment (Eijebong) [inline src/lib.rs]: I don't really like this, I think it takes way too much space.

Comment (lnicola) [reply src/lib.rs]: Neither do I, but I'm not sure it's worth fighting `rustfmt`.

Comment (Eijebong) [reply src/lib.rs]: `#[cfg_attr(rustfmt, rustfmt_skip)]` seems easy enough to me.

Comment (lnicola) [reply src/lib.rs]: Well, that was easier than I thought 😄.

**Pull request merged**